lib/main.coffee

```diff
@@ -10,74 +9,63 @@ module.exports =
     enableAutoActivation:
       title: 'Show Suggestions On Keystroke'
       description: 'Suggestions will show as you type if this preference is enabled. If it is disabled, you can still see suggestions by using the keybinding for autocomplete-plus:activate (shown below).'
-      type: "boolean"
+      type: 'boolean'
       default: true
       order: 1
     autoActivationDelay:
       title: 'Delay Before Suggestions Are Shown'
       description: 'This prevents suggestions from being shown too frequently. Usually, the default works well. A lower value than the default has performance implications, and is not advised.'
-      type: "integer"
+      type: 'integer'
       default: 100
       order: 2
     maxSuggestions:
       title: 'Maximum Suggestions'
       description: 'The list of suggestions will be limited to this number.'
-      type: "integer"
+      type: 'integer'
       default: 10
       order: 3
     confirmCompletion:
       title: 'Keybinding(s) For Confirming A Suggestion'
       description: 'You should use the key(s) indicated here to confirm a suggestion from the suggestion list and have it inserted into the file.'
-      type: "string"
-      default: "tab"
-      enum: ["tab", "enter", "tab and enter"]
+      type: 'string'
+      default: 'tab'
+      enum: ['tab', 'enter', 'tab and enter']
       order: 4
     navigateCompletions:
       title: 'Keybindings For Navigating The Suggestion List'
       description: 'You should use the keys indicated here to select suggestions in the suggestion list (moving up or down).'
-      type: "string"
-      default: "up,down"
-      enum: ["up,down", "ctrl-p,ctrl-n"]
+      type: 'string'
+      default: 'up,down'
+      enum: ['up,down', 'ctrl-p,ctrl-n']
       order: 5
     fileBlacklist:
-      type: "string"
-      default: ".*, *.md"
-      order: 90
+      title: 'File Blacklist'
+      description: 'Suggestions will not be provided for files matching this list.'
+      type: 'string'
+      default: '.*, *.md'
+      order: 6
+    scopeBlacklist:
```

Comment:
I think that just means we need to support blacklisting of the nested scope. I.e. 'source.js comment` (not an actual scope, just pseudocode) could be used to restrict to the combination of the two scopes.